src/patternfly/layouts/Toolbar/examples/toolbar-section-example.hbs

```diff
@@ -0,0 +1,42 @@
+{{#> toolbar}}
+  {{#> toolbar-section aria-label="First section" pf-l-toolbar--modifier="pf-m-align-items-start"}}
+    {{#> toolbar-group newcontent pf-l-toolbar--modifier="pf-m-grow"}}
+      {{#> toolbar-item newcontent}}
+      {{#> title titleType="h3" pf-c-title--Modifier="pf-m-md"}}
+          Some title here
+      {{/title}} 
+        <small>Some supporting text here.</small>
+      {{/toolbar-item}}
+    {{/toolbar-group}}
+    {{#> toolbar-group newcontent pf-l-toolbar--modifier="pf-m-nowrap"}}
+      {{#> toolbar-item newcontent}}
+        {{#> button btnClass="pf-m-action" aria-label="Some action"}}
+          <i class="fas fa-external-link-alt" aria-hidden="true"></i>
+        {{/button}}
+      {{/toolbar-item}}    
+      {{#> toolbar-item newcontent}}
+        {{#> button btnClass="pf-m-action" aria-label="Some action"}}
+          <i class="fas fa-ellipsis-v" aria-hidden="true"></i>
+        {{/button}}
+      {{/toolbar-item}}
+    {{/toolbar-group}}
+  {{/toolbar-section}}
+  {{#> toolbar-section newcontent aria-label="Second section" pf-l-toolbar--modifier="pf-m-wrap pf-m-align-items-baseline"}}
+    {{#> toolbar-group newcontent}}
+      {{#> toolbar-item}}
+        {{#> badge pf-c-badge--modifier="pf-m-read"}}17{{/badge}}
+      {{/toolbar-item}}
+      {{#> toolbar-item newco}}
+        {{#> badge pf-c-badge--modifier="pf-m-unread"}}3{{/badge}}
+      {{/toolbar-item}}
+      {{#> toolbar-item}}
+        {{#> badge pf-c-badge--modifier="pf-m-unread"}}543+{{/badge}}
```

Comment:
Per the spec @mceledonia gave, I think this might be technically correct? Though logically, I would expect that as the value in a badge hits 999, it is represented as `999+` until the value drops below that number again.